lib/libshout-idjc/src/shout.c

```diff
@@ -1308,6 +1308,7 @@ static int try_connect(shout_t *self)
             }
 #endif
             self->state = SHOUT_STATE_REQ_CREATION;
+        /* fall through */
```

Comment:
Why don't we use the c++17 attribute? https://en.cppreference.com/w/cpp/language/attributes/fallthrough
```suggestion
            [[falltrough]];
```

Reply:
Makes sense.